src/pages/contact.js

```diff
@@ -1,84 +1,164 @@
 import React from "react"
 import { Link } from "gatsby"
```

Comment:
I think I forgot about this elsewhere (including the pages I worked on), but can you try including the `Link` component from this plugin:
```
import { Link } from 'gatsby-plugin-intl'
```

It's the library that handles putting `es` and `en` in the path. I think it includes the right one for links too. It may work without it? But... I'm not sure how.